include/deal.II/base/mpi.h

```diff
@@ -1594,9 +1758,9 @@ namespace Utilities
                                 const Iterator  end,
                                 const MPI_Comm &comm)
     {
-      // below we do simple and straight-forward implementation. More elaborate
-      // options are:
-      // http://dx.doi.org/10.1145/2807591.2807644 section 3.1.2
+      // below we do simple and straight-forward implementation. More
+      // elaborate options are: http://dx.doi.org/10.1145/2807591.2807644
+      // section 3.1.2
```

Comment:
I think these changes are spurious.

Reply:
Uhm I did not touch these lines except through `make indent`, should I revert them to the original form?

Reply:
Yes, please revert them.

Reply:
Fixed in 940722e